### PR TITLE
Add Stable Diffusion examples on GPU and CPU

### PR DESCRIPTION
**Pull request by ivy-lv11**
Add Stable Diffusion examples on GPU and CPU

### CODE REVIEW

File: python/llm/example/GPU/StableDiffusion/README.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# Stable Diffusion
+In this directory, you will find examples on how to run StableDiffusion models on [Intel GPUs](../README.md).
+
+### 1. Installation
+#### 1.1 Installation on Linux
+We suggest using conda to manage environment. 
+```bash
+conda create -n diffusion python=3.11
```
Comment (shane-huang) [inline python/llm/example/GPU/StableDiffusion/README.md:8]: remove the conda env creation part but use `conda activate diffusion` instead, as ipex-llm installation part has created it.

File: python/llm/example/CPU/StableDiffusion/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Stable Diffusion
+In this directory, you will find examples on how to run StableDiffusion models on CPU.
+
+### 1. Installation
+#### 1.1 Installation on Linux
```
Comment (shane-huang) [inline python/llm/example/CPU/StableDiffusion/README.md:5]: #### 1.1. Install IPEX-LLM 
Follow the instructions in [IPEX-LLM CPU installation guide](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Overview/install_cpu.html) to install ipex-llm. We recommend to use miniconda to manage your python environment. 

#### 1.2 Install dependencies for Stable Diffusion
Assume you have created a conda environment named `diffusion` with `ipex-llm` installed. Run below commands to install dependencies for running Stable Diffusion.  
```python
conda activate `diffusion`
pip install ...
...
```

**Review by shane-huang — APPROVED** [review]
LGTM

**Comment by shane-huang** [comment]
add one generated picture?

**Comment by ivy-lv11** [comment]
> add one generated picture?

Add figure generated by sdxl-base-1.0

**Pull request merged**